module/os/windows/CMakeLists.txt

```diff
@@ -82,15 +82,15 @@ install(FILES
 
 file(GLOB CSOURCES CONFIGURE_DEPENDS "*.c")
 
-function(set_file_properties)
+function(add_macro_property)
         foreach (I ${CSOURCES})
                 get_filename_component(OUTPUT_FILE_WE ${I} NAME_WE)
                 set_source_files_properties(${I} PROPERTIES COMPILE_FLAGS -DWPPFILE=${CMAKE_SOURCE_DIR}/out/build/${OUTPUT_FILE_WE}.tmh)
                 message(STATUS "FILES_IN ======: ${CMAKE_SOURCE_DIR}/out/build/${OUTPUT_FILE_WE}.tmh") 
         endforeach()
 endfunction()
 
-function(tracewpp OUTPUT_VAR OUTPUT_DIR SOURCE)
+function(tracewpp OUTPUT_DIR SOURCE)
         # [mlr] list of .tmh files to be generated -> TMH
         set(WPP_DIR "C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.19041.0")
```

Comment:
This looks fine, and I think you guys will mostly be using it - so I don't mind either way, but, is there no ENV VAR you can use here instead of hardcoding a "10.0.19041.1" path? It would futureproof it a little for yourselves.

Reply:
Thanks for pointing out

I have updated the PR with ENV VAR